### DISCUSSION TITLE
Is PERMITTED PROCESS allowed in the latest MAC OS Version?

### DISCUSSION THREAD

**Discussion by mufasi4** [SafeExamBrowser/seb-mac#280] [Q&A]
Is PERMITTED PROCESS allowed in the latest MAC OS Version?

please suggest

**Comment by danschlet** [comment]
Running third party applications despite the permitted processes setting already being available is **currently not possible in the macOS version**, as the manual states.

We will add such a feature later this year, although because it will be based on Apple's API in the macOS Assessment Mode (AAC kiosk mode, not the default SEB kiosk mode), it's unclear if this will work also with complex applications like Word and Excel. Apple intends this feature to used with simple tools for example for accessibility purposes. **Also the feature doesn't stop applications from ACCESSING THE FILE SYSTEM!** So don't expect this to allow perfectly secure exams.

See this discussion threads:
https://github.com/SafeExamBrowser/seb-mac/discussions/114
https://github.com/SafeExamBrowser/seb-mac/discussions/276
